dataplane/forwarding/fwdport/ports/kernel.go

```diff
@@ -67,6 +71,7 @@ func (p *kernelPort) Cleanup() {
 	p.input.Cleanup()
 	p.output.Cleanup()
 	p.handle.Close()
+	close(p.linkDoneCh)
```

Comment:
close(p.linkUpdateCh) as well?

Reply:
sending on closed channel panics, so relying on the GC so close the channel once all goroutines are done with it. (not that it really matter in this case)